eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -131,16 +131,18 @@ public static function installWelcomeText(Event $event)
 
 <fg=cyan>Welcome to ${installName}!</fg=cyan>
 
-<options=bold>Quick dev/test install:</>
+<options=bold>Quick installation to test in local dev environment:</>
 <comment>    $  export SYMFONY_ENV="dev"</comment>
 ${installCommandText}
 <comment>    $  php ${consoleDir}/console server:run</comment>
 
 Note:
-- Instructions above assume the CLI user you execute these commands with is the same one that extracted/installed the software.
-- The last command will give you the url to the frontend of the installation, add "/admin" to reach backend.
+- Instructions assume CLI user you execute commands with is the same one that extracted/installed the software.
```

Comment:
Shorter? Yes we can ;)

```suggestion
- The instructions assume you execute commands with the CLI user that extracted/installed the software.
```